enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -113,7 +113,7 @@ <h2 class="consent-title">{{ consent_message_header }}</h2>
                     <header class="modal-header">
                       <h2 id="modal-header-text">{{ confirmation_modal_header }}</h2>
                     </header>
-                    <p>{{ confirmation_alert_prompt }}</p>
+                    <p>{{ confirmation_alert_prompt | safe }}</p>
```

Comment:
We want to remove the spaces here to the left/right of the pipe, both here and above.
```
<p>{{ confirmation_alert_prompt|safe }}</p>
```